src/python/grpcio/grpc/_channel.py

```diff
@@ -1439,15 +1588,19 @@ def _unsubscribe(state, callback):
                 break
 
 
-def _augment_options(base_options, compression):
+def _augment_options(
+        base_options: Optional[ChannelArgumentType],
+        compression: Optional[grpc.Compression]) -> Sequence[Tuple[str, Any]]:
     compression_option = _compression.create_channel_option(compression)
     return tuple(base_options) + compression_option + ((
         cygrpc.ChannelArgKey.primary_user_agent_string,
         _USER_AGENT,
     ),)
 
 
-def _separate_channel_options(options):
+def _separate_channel_options(
+    options: Optional[ChannelArgumentType]
```

Comment:
Why `Optional`?

Reply:
Took a looks gain, `options` is required here. (This is the same `options` passed from channel creation (`Channel.__init__`), the doc string didn't mention it can be `None` or `Optional`).

PS: Glad we're adding types so we don't need manually search for keywords like `optional` or `None`.